modules/adbutlerBidAdapter.js

```diff
@@ -1,130 +1,122 @@
-/**
- * @overview AdButler Prebid.js adapter.
- * @author dkharton
- */
-
 'use strict';
 
-var utils = require('src/utils.js');
-var adloader = require('src/adloader.js');
-var bidmanager = require('src/bidmanager.js');
-var bidfactory = require('src/bidfactory.js');
-var adaptermanager = require('src/adaptermanager');
-
-var AdButlerAdapter = function AdButlerAdapter() {
-  function _callBids(params) {
-    var bids = params.bids || [];
-    var callbackData = {};
-    var zoneCount = {};
-    var pageID = Math.floor(Math.random() * 10e6);
-
-    // Build and send bid requests
-    for (var i = 0; i < bids.length; i++) {
-      var bid = bids[i];
-      var zoneID = utils.getBidIdParameter('zoneID', bid.params);
-      var callbackID;
-
-      if (!(zoneID in zoneCount)) {
-        zoneCount[zoneID] = 0;
+import * as utils from 'src/utils';
+import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'adbutler';
+
+export const spec = {
+  code: BIDDER_CODE,
+  pageID: Math.floor(Math.random() * 10e6),
+  zoneCounters: {},
```

Comment:
No mutable globals allowed... see [the 1.0 conventions](http://prebid.org/dev-docs/bidder-adapter-1.html#required-adapter-conventions).

Can this be moved inside your `buildRequests` function?